feature-libs/checkout/b2b/ng-package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "$schema": "../../../node_modules/ng-packagr/ng-package.schema.json",
+  "lib": {
+    "entryFile": "./public_api.ts",
+    "umdModuleIds": {}
```

Comment:
please remove

I guess it's the same case as base where it didn't throw/warn anything